app/facades/food_search_facade.rb

```diff
@@ -0,0 +1,22 @@
+class FoodSearchFacade
+
+  def self.search_foods(query)
+    results = service.get_foods(query)
+    poros = results[:foods].map { |data| Food.new(data) }
+
+    foods = []
+    poros.each do |food|
+      if food.ingredients != nil
+        if food.ingredients.include? query.upcase
+          foods << food
+        end
+      end
```

Comment:
This is an interesting interpretation of the user story! It totally works and it also would've been fine if you opted to just return the results as is even if there would be some gaps in the data.